src/ApiService/Cart/addCart.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const addCart = async (product, token) => {
+  try {
+    const { data, status } = await axios.post(
+      "api/user/cart",
+      { product },
+      {
+        headers: {
+          authorization: token,
+        },
+      }
+    );
+    if (status === 200 || status === 201) {
+      alert("Product added into cart");
```

Comment:
no need for alert as Text changes to ```go to cart`` signifies addition

Reply:
remove as said . thanks for pointing out this